### ISSUE TITLE
Weight Loss Plan: The Goal to Go For

### ISSUE THREAD

**Issue by nadjibandnaaima** [nadjibandnaaima/diets-for-quick-weight-loss#978]
<div dir="ltr" style="text-align: left;">
<i><b><span style="font-size: x-large;">Weight Loss Plan: The Goal to Go For</span></b></i><br>
<ins class="adsbygoogle" style="display: inline-block; width: 300px; height: 600px;" data-ad-client="ca-pub-2862230082201059" data-ad-slot="1546920322"></ins> 
//<![CDATA[
(adsbygoogle = window.adsbygoogle || []).push({});
//]]>
<br>
Since excess weight puts you at risk for many health problems, you may need to set some weight loss plans to help avoid those risks and prevent disease.<br><br>
But what should be your long-term goal? And what short-term goals should you set to help you get there? You have a better chance of attaining your goals if you make sure that the weight loss plans that you will use are sensible and reasonable right at the beginning.<br><br>
Here are some guidelines from the experts in choosing weight loss plans and goals.<br><br>
1. Be realistic<br><br>
Most people&rsquo;s long-term weight loss plans are more ambitious than they have to be.<br><br>
For example, if you weigh 170 pounds and your long-term plan is to weigh 120, even if you have not weighed 120 since you were 16 and now you are 45, that is not a realistic weight loss goal.<br><br>
Your body mass index or BMI is a good indicator of whether or not you need to shed of pounds. The ideal BMI range, according to the national Institutes of Health, is between 19 and 24.9. If your BMI is between 25 and 29.9, you are considered overweight. Any number above 30 is in the obesity range.<br><br>
From this point of view, you will need a sensible weight loss plan that will correspond to the required BMI based on your height, because this is the primary factor that will affect your BMI.<br><br>
2. Set appropriate objectives<br><br>
Using a weight loss plan just for vanity&rsquo;s sake is psychologically less helpful than losing weight to improve health.<br><br>
You have made a big step forward if you decide to undergo a weight loss plan that includes exercise and eating right so that you will feel better and have more energy to do something positive in your life.<br><br>
3. Focus on doing, not losing<br><br>
Rather than saying that you are going to lose a pound this week, say how much you are going to exercise this week. This would definitely make up of a sensible weight loss plan.<br><br>
Keep in mind that your weight within a span of a week is not completely in your control, but your behavior is.<br><br>
4. Build bit by bit<br><br>
Short-term weight loss plans should not be &ldquo;pie-in-the-sky.&rdquo; This means that when you have never exercised at all, your best weight loss plan for this week should be based on finding three different one-mile routes that you can walk next week.<br><br>
5. Keep up the self-encouragement<br><br>
An all-or-nothing attitude only sets you up to fail. Learn to evaluate your efforts fairly and objectively. If you fall short of some goals, just look ahead to next week. You do not need to have a perfect record.<br><br>
After all, self-encouragement should definitely be a part of your weight loss plans. Otherwise, you will just fail in the end.<br><br>
6. Use measurable measures<br><br>
Saying that you are going to be more positive this week or that you are going to really get serious this week is not a goal that you can measure and should not be a part of your weight loss plan.<br><br>
This is another reason why you should incorporate exercise on your weight loss plan and focus on it. You should be able to count up the minutes of exercise in order to be successful in your plan.<br><br>
The bottom line is, people should make weight loss plans that will only remain as it is, just a plan. They have to put it into action by incorporating goals that will motivate them to succeed.<br>
</div>
<br><br>
via diets for quick weight loss http://ift.tt/1SU2WKt<br>
January 16, 2016 at 02:27PM
